### ISSUE TITLE
关于densenet训练阶段BN层的设置

### ISSUE THREAD

**Issue by TongYoungG** [shicai/DenseNet-Caffe#34]
您好，在densenet训练阶段，train.prototxt中是否要设置BN层在TRAIN和TEST两种phase下的参数use_global_stats的状态，例如这样：
layer {
	bottom: "conv6"
	top: "conv6"
	name: "bn_conv6"
	type: "BatchNorm"
	batch_norm_param {
		use_global_stats: false
	}
    include {
        phase: TRAIN
    }
}
layer {
    bottom: "conv6"
    top: "conv6"
    name: "bn_conv6"
    type: "BatchNorm"
    batch_norm_param {
        use_global_stats: true
    }
    include{
        phase: TEST
    }
} 
还是说只需要设置一种状态，然后在deploy.prototxt中，设置use_global_stats=true就行了

**Comment by sunjunlishi** [comment]
应该自己会判断的吧。不要搞这个项。对了 我的loss 怎么是 nan；
有遇到过？
